contracts/env/.env.arbitrum-sepolia

```diff
@@ -11,4 +11,4 @@ ATTESTATION_REGISTRY_ADDRESS=0xee5e23492bf49C1F4CF0676b3bF49d78A6dD61c5
 MODULE_REGISTRY_ADDRESS=0xEC572277d4E87a64DcfA774ED219Dd4E69E4BDc6
 PORTAL_REGISTRY_ADDRESS=0x1ceb52584B6C45C7049dc7fDC476bC138E4beaDE
 SCHEMA_REGISTRY_ADDRESS=0x025531b655D9EE335B8E6cc4C118b313f26ACc8F
-ATTESTATION_READER_ADDRESS=0xBdC45324AB9A7e82Ae15324a3d8352b513Ee2788
+ATTESTATION_READER_ADDRESS=
```

Comment:
Is it supposed to have a value?

Reply:
No, because there is no EAS instance on Arbitrum Sepolia (I didn't notice it before).  
So there is no point in having an `AttestationReader` instance on this network.